docker-compose.yml

```diff
@@ -2,47 +2,45 @@
 version: '3'
 
 services:
-
   # Main application
   app:
-    build: devops/docker/tasking-manager
+    image: hotosm/tasking-manager
+    container_name: app
     env_file: tasking-manager.env
+    environment:
+      - POSTGRES_ENDPOINT=postgresql
     depends_on:
       - postgresql
     ports:
       - "5000:5000"
+      - "3000:3000"
```

Comment:
Why `3000` instead of `5000`?

Reply:
Port 3000 should be included with the purpose of client development using gulp. I moved it to the override file.

Reply:
Cool. Can we use `3000` as the one we expose from the Dockerfile by default? Or is there a reason we want to use `5000`?

Reply:
**EXPOSE** will enable communication between containers that belong to different docker networks. I think you are referring to establish communication between the container and the host machine. For this case, **PUBLISH** will be the command. IMO, publishing the 3000 for development would be ok when creating the docker instances. For production, it would involve security issues. Let me know your opinion.

Reply:
I just want to be consistent. And using `5000` at some places and `3000` on others will most likely cause problems and confusion. Especially for new ones. Is there a reason we want to use both?

Reply:
Never mind, I was still looking at yesterdays PR proposal. The new changes make sense of course.